src/ui/public/courier/fetch/fetch_now.js

```diff
@@ -30,7 +30,7 @@ export function FetchNowProvider(Private, Promise) {
       if (!req.started) return req;
       return req.retry();
     }))
-      .catch(courierNotifier.fatal);
+      .catch(error => fatalError(error, location));
```

Comment:
seems like the optimizer is tripping over the uninitialized location variable

Reply:
Oh shoot! Good catch, thanks. We should add a linting rule for that!